### ISSUE TITLE
aido1_remote3-v3 protocol support

### ISSUE THREAD

**Issue by AndreaCensi** [duckietown/challenge-aido1_LF1-template-pytorch#1]
Currently the template does not support the protocol aido1_remote3-v3.

I started working on this on the branch aido1_remote3-v3.

Note that you have to pin the corresponding branches in aido1_remote3-v3:
```
# pin these to aido1_remote3-v3
-e git://github.com/duckietown/gym-duckietown-agent.git@aido1_remote3-v3#egg=gym-duckietown-agent
-e git+https://github.com/duckietown/duckietown-slimremote.git@aido1_remote3-v3#egg=duckietown-slimremote
```

**Comment by AndreaCensi** [comment]
Current state - if one runs the submission the following error is shown:
```
evaluator_1  | Traceback (most recent call last):
evaluator_1  |   File "/workspace/src/duckietown-challenges/src/duckietown_challenges/cie_concrete.py", line 581, in wrap_solution
evaluator_1  |     solution.run(cis)
evaluator_1  |   File "solution.py", line 87, in run
evaluator_1  |     solve(params, cis)
evaluator_1  |   File "solution.py", line 60, in solve
evaluator_1  |     action = model.predict(observation)
evaluator_1  |   File "/workspace/model.py", line 172, in predict
evaluator_1  |     assert state.shape[1] == 120
evaluator_1  | AssertionError
evaluator_1  |
```

**Comment by bhairavmehta95** [comment]
Fixed in #3. Closing
